src/client/cmd/launch.cpp

```diff
@@ -207,7 +194,7 @@ mp::ReturnCode cmd::Launch::request_launch()
 
         if (reply.metrics_pending())
         {
-            if (mcp::is_tty())
+            if (term->cin_is_live())
```

Comment:
Should this check both cin and cout since there is both output and input?  Maybe another helper function in the Terminal class that checks both at the same time?

Although previously we only checked if stdin is a tty, so I'm not sure how much it matters...

Reply:
I had similar thought, but was concerned in case you knew something I didn't, so left it. I've done as you asked.